test/job-test.js

```diff
@@ -246,6 +247,71 @@ module.exports = {
           }, 1000);
         }*/
   },
+  "#schedule(RRule)": {
+    "Runs job at interval based on rrule, repeating indefinitely": function(test) {
+      test.expect(3);
+
+      var job = new schedule.Job(function() {
+        test.ok(true);
+      });
+
+      var rrule = new RRule({
+        freq: RRule.SECONDLY
+      });
+
+      job.schedule(rrule);
+
+      setTimeout(function() {
+        job.cancel();
+        test.done();
+      }, 3250);
+
+      clock.tick(3250);
+    },
+    "Job emits 'scheduled' event for every next invocation": function(test) {
+        // Job will run 3 times but be scheduled 4 times, 4th run never happens
+        // due to cancel.
+        test.expect(4);
+
+        var job = new schedule.Job(function() {});
+
+        job.on('scheduled', function(runOnDate) {
+          test.ok(true);
+        });
+
+        var rrule = new RRule({
+          freq: RRule.SECONDLY
+        });
+
+        job.schedule(rrule);
+
+        setTimeout(function() {
+          job.cancel();
+          test.done();
+        }, 3250);
+
+        clock.tick(3250);
+      }
+      /*,
```

Comment:
Why is this commented out?

Reply:
Cant remember I wrote it a while ago and havent come back to it until now, ill get back to you on this